tools/sextant/src/inspect.rs

```diff
@@ -141,12 +142,12 @@ mounts:
       type: bind
       host: /system";
 
-    fn create_test_npk(dest: &Path) -> PathBuf {
+    fn create_test_npk(dest: &PathBuf) -> PathBuf {
         let src = create_tmp_dir();
         let key_dir = create_tmp_dir();
-        let manifest = create_test_manifest(&src);
-        let (_pub_key, prv_key) = gen_test_key(&key_dir);
-        pack(&manifest, &src, &dest, Some(&prv_key)).expect("Pack NPK");
+        let manifest = create_test_manifest(&src.path().to_path_buf());
+        let (_pub_key, prv_key) = gen_test_key(&key_dir.path().to_path_buf());
+        pack(&manifest, &src.path().to_path_buf(), &dest, Some(&prv_key)).expect("Pack NPK");
```

Comment:
`pack` takes a `&Path`. Why creating a `PathBuf` and converting it to `&Path` again? Think `src.path()` should be fine because of https://docs.rs/tempfile/3.2.0/tempfile/struct.TempDir.html#method.path already returns `&Path`.